src/config/src/config.erl

```diff
@@ -455,6 +466,24 @@ read_ini_file(IniFile) ->
             throw({startup_error, Msg})
     end.
 
+get_write_file([_ | _] = IniFiles) ->
+    lists:last(IniFiles);
+get_write_file(_) ->
+    undefined.
+
+% Takes a list of files and directories with *.ini files, and expands
+% the directories inline with individual *.ini files
+expand_dirs(IniFilesDirs) ->
+    lists:flatmap(fun expand_dirs_fmap/1, IniFilesDirs).
+
+expand_dirs_fmap(File) ->
+    case filelib:is_dir(File) of
+        true ->
+            lists:sort(filelib:wildcard(File ++ "/*.ini"));
```

Comment:
let's switch to the safer `wildcard("*.ini", File)` construction here while we're at it.

Reply:
@nickva ^^

Reply:
that's a safer option, agreed, I'll update it

Reply:
Updated. One difference was also that wildcard/2 doesn't produce full paths just plain filenames, so to return the previous results we have to prepend the directory to the result:

```erlang
            Fun = fun(IniFile) -> filename:join(File, IniFile) end,
            lists:map(Fun, lists:sort(filelib:wildcard("*.ini", File)));
```